private/protocol/timestamp.go

```diff
@@ -62,10 +68,12 @@ func ParseTime(formatName, value string) (time.Time, error) {
 		return time.Parse(ISO8601TimeFormat, value)
 	case UnixTimeFormatName:
 		v, err := strconv.ParseFloat(value, 64)
+		_, dec := math.Modf(v)
+		dec = math.Round(dec*1e3)/1e3 //Rounds 0.1229999 to 0.123
```

Comment:
Is rounding the decimal needed?

Reply:
The [math.Round](https://golang.org/pkg/math/#Round) is also what is causing the TravisCI tests to fail, because `math.Round` wasn't added to the `math` package until Go 1.10, and the SDK supports back to Go 1.5.

Reply:
"strconv.ParseFloat()" looses precision after some decimals, so the TestParseTime considers input and output equal if they are equal up to 3 decimals.
Should I do the rounding without using math.Round?